lib/utils/compose.js

```diff
@@ -443,7 +443,10 @@ export class BuildProgressUI {
 		this._startTime = Date.now();
 	}
 
-	end(summary = null) {
+	/**
+	 * @param {Dictionary<string> | undefined} summary
```

Comment:
This is to make it match the typing it ends up getting assigned to, the typescript update recognized that they were different (presumably it was defaulting to `any` before)